### ISSUE TITLE
Error dialog after application start ('Nuvola.parseTimeUsec' is undefined)

### ISSUE THREAD

**Issue by IgorKvasn** [tiliado/nuvola-app-deezer#11]
When I start Nuvola Deezer, I get error dialog with message:
```
Failed to inject JavaScript API. Function 'Nuvola.core.emit' failed. file:///usr/share/nuvolaplayer3/web_apps/deezer/integrate.js:139: TypeError: Nuvola.parseTimeUsec is not a function. (In 'Nuvola.parseTimeUsec(elm ? elm.innerText || null : null)', 'Nuvola.parseTimeUsec' is undefined)

The application might not function properly.
```

Keyboard shortcuts don't work as well as play/pause, next/previous track actions in tray icon don't work (they are disabled).

Nuvola Player 3 3.0.8
Revision 683fc7254443f13a35e4ba33d948ca6e382e690f
WebKitGTK 2.16.3
libsoup 2.52.2

Deezer Nuvola version: 2.6
OS: Linux Mint 18.1

**Comment by fenryxo** [comment]
Hello @IgorKvasn. Thanks for reporting the issue. I'll fix it as soon as possible.

**Comment by fenryxo** [comment]
Fixed in Deezer script 2.7.
